core/domain/exp_services_test.py

```diff
@@ -1673,6 +1682,17 @@ class ExplorationCommitLogUnitTests(ExplorationServicesUnitTests):
         'post_commit_status': 'public'
     }
 
+    COMMIT_UPDATE_FIRST_PUBLISHED_MSEC_EXP_2 = {
```

Comment:
I don't think I understand why this is actually a new commit. It seems like a weird thing to put in the exploration history...